Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -334,5 +325,33 @@ void UpdateLayout()
 			Layout();
 #endif
 		}
+
+		void UpdateMaxLines()
+		{
+			if (Element.MaxLines >= 0)
+			{
+#if __MOBILE__
+				Control.Lines = Element.MaxLines;
+
+				LayoutSubviews();
+#else
+				Control.MaximumNumberOfLines = Element.MaxLines;
+
+				Layout();
+#endif
+			}
+			else
+			{
+#if __MOBILE__
+				Control.Lines = 1;
+
+				LayoutSubviews();
```

Comment:
it probably depends on the LineBreakMode

Reply:
Default value == 1, as per: https://developer.apple.com/documentation/uikit/uilabel/1620539-numberoflines

Reply:
Default value == 1, as per: https://developer.apple.com/documentation/uikit/uilabel/1620539-numberoflines.

But agreed that using in conjunction with line break mode makes sense. Then 1 for truncation and 0 (which acts as auto) for wrapping?

Reply:
> Then 1 for truncation and 0 (which acts as auto) for wrapping?

that's the way it was, iirc

Reply:
You're right! 🙂 pfew, have to get back in there. Was a while ago when I drafted this